tests/schema/test_generics_nested.py

```diff
@@ -259,9 +259,9 @@ def blocks(self) -> List[BlockInterface]:
         }
 
         """
-        The `JSON` scalar type represents JSON values as specified by [ECMA-404](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-404.pdf).
+        The `JSON` scalar type represents JSON values as specified by [ECMA-404](https://ecma-international.org/wp-content/uploads/ECMA-404_2nd_edition_december_2017.pdf).
```

Comment:
can you rebase? I think this was fixed recently

Reply:
You were right.  I merged main and this change went away.